src/main/kotlin/no/nav/tilleggsstonader/sak/vilkår/stønadsvilkår/dto/json/VilkårJson.kt

```diff
@@ -0,0 +1,111 @@
+package no.nav.tilleggsstonader.sak.vilkår.stønadsvilkår.dto.json
+
+import no.nav.tilleggsstonader.sak.vilkår.stønadsvilkår.domain.Vilkår
+import no.nav.tilleggsstonader.sak.vilkår.stønadsvilkår.domain.VilkårType
+import no.nav.tilleggsstonader.sak.vilkår.stønadsvilkår.domain.Vilkårsresultat
+import no.nav.tilleggsstonader.sak.vilkår.stønadsvilkår.dto.OpphavsvilkårDto
+import no.nav.tilleggsstonader.sak.vilkår.stønadsvilkår.dto.VilkårDto
+import no.nav.tilleggsstonader.sak.vilkår.stønadsvilkår.dto.VurderingDto
+import no.nav.tilleggsstonader.sak.vilkår.stønadsvilkår.dto.tilDto
+import no.nav.tilleggsstonader.sak.vilkår.stønadsvilkår.regler.BegrunnelseType
+import no.nav.tilleggsstonader.sak.vilkår.stønadsvilkår.regler.RegelId
+import no.nav.tilleggsstonader.sak.vilkår.stønadsvilkår.regler.SvarId
+import no.nav.tilleggsstonader.sak.vilkår.stønadsvilkår.regler.SvarRegel
+import no.nav.tilleggsstonader.sak.vilkår.stønadsvilkår.vilkårsreglerPassBarn
+import java.time.LocalDateTime
+import java.util.UUID
+
+data class VilkårJson(
+    val id: UUID,
+    val behandlingId: UUID,
+    val resultat: Vilkårsresultat,
+    val vilkårType: VilkårType,
+    val barnId: UUID? = null,
+    val endretAv: String,
+    val endretTid: LocalDateTime,
+    val delvilkårsett: DelvilkårsettJson,
+    val opphavsvilkår: OpphavsvilkårDto?,
+)
+
+private typealias DelvilkårsettJson = Map<RegelId, DelvilkårJson>
+
+data class DelvilkårJson(
+    val svar: SvarId? = null,
+    val begrunnelse: String? = null,
+    val svaralternativer: Map<SvarId, SvaralternativJson>,
+    val følgerFraOverordnetValg: OverordnetValgJson? = null,
```

Comment:
Hva er `følgerFraOverordnetValg`?

Reply:
Dersom dette delvilkåret er _avhengig av svaret på et annet delvilkår_, så vil dette feltet ha informasjon om dette. Ellers er feltet null. Veldig mottakelig for navneforslag! :)

Reply:
Etter å ha sjekket frontend så tolker jeg at dette/[min andre kommentar](https://github.com/navikt/tilleggsstonader-sak/pull/244#discussion_r1534721400) at man går fra å peke bakover. Dvs `spm2 -> spm1`
Trengs det? Eller er det tilstrekkelig å kun ha peiling på at `spm1 -> spm 2`?